src/screens/welcome/install-referrer/androidPlayInstallReferrer.ts

```diff
@@ -0,0 +1,41 @@
+import Minilog from '@cozy/minilog'
+import {
+  PlayInstallReferrer,
+  PlayInstallReferrerInfo
+} from 'react-native-play-install-referrer'
+
+import {
+  getEnforcedInstallReferrer,
+  shouldForceInstallReferrer
+} from '/core/tools/env'
+
+const log = Minilog('AndroidPlayInstallReferrer')
+
+export const getInstallReferrer =
+  (): Promise<PlayInstallReferrerInfo | null> => {
+    if (shouldForceInstallReferrer()) {
+      const installReferrer = getEnforcedInstallReferrer()
+      log.debug('Enforce InstallReferrer with: ' + installReferrer)
+      return Promise.resolve({
+        installReferrer
+      } as PlayInstallReferrerInfo)
+    }
+
+    return new Promise(resolve => {
+      PlayInstallReferrer.getInstallReferrerInfo(
+        (installReferrerInfo, error): void => {
+          if (!error) {
+            resolve(installReferrerInfo)
+          } else {
+            // Here we silence the error as non-google devices would trigger it
```

Comment:
I would appreciate feedback on this specific choice I made (silencing error and sentry). It that choice OK for you?